### PR TITLE
fix obj textures

### PR DESCRIPTION
**Pull request by sethalves**
- fix a bug that could cause obj textures to be flipped around the wrong way

### CODE REVIEW

**Comment by sethalves** [comment]
http://stackoverflow.com/questions/22616986/order-of-evaluation-of-arguments-in-function-calling

**Comment by sethalves** [comment]
to test:
- load an obj model with a text
- make sure the texture lines up correctly

File: libraries/fbx/src/OBJReader.cpp [hunk 1]
```diff
@@ -123,7 +123,9 @@ glm::vec3 OBJTokenizer::getVec3() {
     return v;
 }
 glm::vec2 OBJTokenizer::getVec2() {
-    auto v = glm::vec2(getFloat(), 1.0f - getFloat());  // OBJ has an odd sense of u, v. Also N.B.: getFloat() has side-effect
+    float uCoord = getFloat();
+    float vCoord = 1.0f - getFloat();
+    auto v = glm::vec2(uCoord, vCoord);
```
Comment (huffman) [inline libraries/fbx/src/OBJReader.cpp]: I must be missing something - doesn't this end up being the same?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 43536e0046963028be210749b32416a849f78aaa available at: https://gist.github.com/7c1df95ff6f00ddf70782afedbe302c8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8955-43536e0.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8955-43536e0.dmg

File: libraries/fbx/src/OBJReader.cpp (hunk 1 above)
Comment (huffman) [inline libraries/fbx/src/OBJReader.cpp]: Ah, got it, thanks!

Comment (sethalves) [inline libraries/fbx/src/OBJReader.cpp]: getFloat has the side-effect of moving the lexer forward.  If the compiler chooses to evaluate 1.0-f -getFloat() before getFloat(), it ends up swapping the u and v coordinates.

Comment (huffman) [inline libraries/fbx/src/OBJReader.cpp]: (I should have read the commit message)

**Comment by ZappoMan** [comment]
do we have a test plan for this?

**Comment by cozza13** [comment]
tested with an OBJ and multiple testures 

capture

**Comment by cozza13** [comment]
Currenty Master

capture

**Pull request merged**